.github/workflows/ci.yml

```diff
@@ -475,9 +475,20 @@ jobs:
           key: cargo-firmware-bins-cache-cachebust-2-${{ hashFiles('./firmware-binaries/Cargo.lock') }}
 
       - name: Building Examples
-        run: cargo build --release
+        run: |
+          export CARGO_TARGET_DIR=../_build/cargo
```

Comment:
It's possible to set the target directory per workspace in the `<workspace-dir>/.cargo/config` file like
```toml
[build]
target-dir = "<path-here>"
```
Maybe that's better because then *all* Cargo commands use that and not only those which have the right environment variable set?